programs/openbook-v2/src/logs.rs

```diff
@@ -83,6 +83,8 @@ pub struct OpenOrdersPositionLog {
     pub market: Pubkey,
     /// Base lots in open bids
     pub bids_base_lots: i64,
+    /// Quote lots in open bids
```

Comment:
please note that this will break compatibility with old events, not sure if you want to create a v2

Reply:
I don't think anyone will be tracking open orders accounts being used until now with these events. We should tell the few people who have currently ooa to close and reopen